### PR TITLE
chore(#16): Postman

### PR DESCRIPTION
**Pull request by fabioalmeida100**
**What was done in this PR?**

- This PR implementation the documentation of API
- Added the public documentation in Postman
- Added Postman Collection for test API in local machine

**Plus:** improved tips for help who want use Docker Setup and changed the suggestion of bind port in **docker run** command.

This PR close the issue #16.

### CODE REVIEW

File: postman_collection/StackOverflowClone - API.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,520 @@
+{
```
Comment (Mayank0255) [inline postman_collection/StackOverflowClone - API.postman_collection.json:1]: @fabioalmeida100 I think the name of the file can just be `API.postman_collection.json`, the StackOverflowClone part can be removed

**Comment by Mayank0255** [comment]
@fabioalmeida100 I ran the updated commands and I got this,

```
C:\Users\Mayank\Desktop\Stackoverflow-Clone-Backend>docker run -p -d 5000:5000 stackoverflowclone
docker: Invalid containerPort: -d.
See 'docker run --help'.
```

And my `.env`, consists of this
```
HOST = 'localhost'
USER = 'root'
DATABASE = 'stack_overflow_v2'
PASSWORD = `MY_PWD`
```

And the db `stack_overflow_v2` is already built in my local mysql with the seed entries

**Comment by fabioalmeida100** [comment]
> @fabioalmeida100 I ran the updated commands and I got this,
> 
> ```
> C:\Users\Mayank\Desktop\Stackoverflow-Clone-Backend>docker run -p -d 5000:5000 stackoverflowclone
> docker: Invalid containerPort: -d.
> See 'docker run --help'.
> ```
> 
> And my `.env`, consists of this
> 
> ```
> HOST = 'localhost'
> USER = 'root'
> DATABASE = 'stack_overflow_v2'
> PASSWORD = `MY_PWD`
> ```
> 
> And the db `stack_overflow_v2` is already built in my local mysql with the seed entries

TLDR: change HOST = 'localhost' to HOST = host.docker.internal and rebuild your image and run your container "docker run -d -p 5000:5000 stackoverflowclone"

**Long answer:** maybe you need to understand how does work docker network. Please, read documentation about it attached in link README.md.

You is trying to connect the database with **localhost**. Who is localhost inside of container? It is the own container, and it doesn't work for your environment (Windows and database in host machine).

In relation your error: the command was typed wrong in README.md and I fixed it. But, remember: Docker Setup is good for who have some experience with Docker and commands samples are only a suggestion of setup. In real life you can do it bind your choose port, with docker compose or not... Much possibilities and I wrote here a little suggestion ;-)

File: postman_collection/StackOverflowClone - API.postman_collection.json (hunk 1 above)
Comment (fabioalmeida100) [reply postman_collection/StackOverflowClone - API.postman_collection.json]: Done

**Comment by Mayank0255** [comment]
@fabioalmeida100 Merged the PR, sorry for the delay, got really busy.
